.pre-commit-config.yaml

```diff
@@ -30,13 +30,21 @@ repos:
     rev: v3.1.0
     hooks:
       - id: prettier
+  - repo: local
+    hooks:
+      - id: tsc
+        name: tsc
+        entry: "pre-commit-ecr-viewer-tsc.sh"
+        language: script
+        pass_filenames: false
+
 ci:
   autofix_commit_msg: |
     [pre-commit.ci] auto fixes from pre-commit hooks
   autofix_prs: true
   autoupdate_commit_msg: "[pre-commit.ci] pre-commit autoupdate"
   autoupdate_schedule: monthly
-  skip: []
+  skip: [tsc] # needs install, plus already covered by other ci
```

Comment:
Is the other CI you are refering to the [javascript-linting job](https://github.com/CDCgov/dibbs-ecr-viewer/blob/main/.github/workflows/container-ecr-viewer.yaml#L30-L38)? If so can you add `npm run lint:tsc` in there too because [currently it is  not running it](https://github.com/CDCgov/dibbs-ecr-viewer/actions/runs/13545900235/job/37857328976).

Reply:
thanks! I was thinking more generally that the build fails if there are typing problems, which doesn't catch the test. This is much better. Added it over there